### PR TITLE
Standardize report names to use underscores

### LINKED ISSUE #5110: Standardize the way we name reports
In working on https://github.com/IMA-WorldHealth/bhima/issues/5108, I realized that we use three forms of naming our reports:
  1. `camelCase`
  2. `snake-case`
  3. `underscore_case`.

This is kinda confusing in the long run and it makes me confused about how I should name my reports.  Let's pick one and stick to it.  I prefer either `snake-case` or `underscore_case` since it isn't case sensitive, but I don't really care which one we use.

My initial vote goes to `snake-case` because it looks like our components.  @mbayopanda @lomamech @jmcameron what do you think?

### PR DESCRIPTION
**Pull request by jmcameron**
Updated and refactor many reports to us a common lowercase_underscore naming convention.

Suggestions for testing

- Basic tests can be done with any site.  However, if bhima_test is not used, make sure to apply the migrations file
- 'yarn build' to rebuild menus, etc
- 'yarn build:db' (for bhima_test) or install suitable site database data and apply migrations file
- Clear the cache in your browser (THIS IS IMPORTANT! Don't skip this)
- yarn dev
- Check the following changed reports (suggest executing each one).  NOTE: It may be easiest to try every report in the system.
   - analysis_auxiliary_cashboxes
   - annual_clients_report
   - break_even
   - break_even_fee_center
   - cashflow_by_service
   - client_debts
   - client_support
   - data_kit
   - debtor_summary
   - employee_standing
   - fee_center
   - financial_patient
   - indicators_report
   - inventory_changes
   - invoiced_received_stock
   - monthly_balance
   - monthly_consumption_report
   - patient_standing
   - purchase_order_analysis
   - realized_profit
   - recovery_capacity
   - system_usage_stat
   - unpaid_invoice_payments

### CODE REVIEW

**Comment by jmcameron** [comment]
Ok, Tomorrow I'll go back and fix the funky naming to hush up hound.

File: client/src/i18n/en/patient_reg.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "PATIENT_REG": {
     "CARD": "Patient Card",
     "ENTITY" : "Patient",
-    "FICHE" : "Patient Fiche",
+    "FICHE" : "Patient Information",
```
Comment (jniles) [inline client/src/i18n/en/patient_reg.json:5]: 👍

File: client/src/modules/employees/templates/action.cell.html [hunk 2]
```diff
@@ -18,7 +18,7 @@
       </a>
     </li>
     <li>
-      <a data-method="financialActivities" ng-href="reports/finance/financialPatient/{{row.entity.patient_uuid}}?lang={{grid.appScope.languageKey}}" target="_blank">
+      <a data-method="financialActivities" ng-href="reports/finance/financial_patient/{{row.entity.patient_uuid}}?lang={{grid.appScope.languageKey}}" target="_blank">
```
Comment (jniles) [inline client/src/modules/employees/templates/action.cell.html:21]: Wow, you did a ton of work here!  Excellent.

**Comment by jmcameron** [comment]
@jniles and @lomamech This PR is now ready to review.

File: client/src/modules/patients/record/bh-patient-financial-activity.html [hunk 3]
```diff
@@ -4,7 +4,7 @@
     <span translate>FORM.LABELS.FINANCIAL_STATUS</span>
     <span class="pull-right">
       <bh-pdf-link
-        pdf-url="/reports/finance/financialPatient/{{ ::$ctrl.patientUuid }}"
+        pdf-url="/reports/finance/financial_patient/{{ ::$ctrl.patientUuid }}"
```
Comment (jniles) [inline client/src/modules/patients/record/bh-patient-financial-activity.html:7]: Excellent 👍

File: server/models/migrations/next/migrate.sql [hunk 4]
```diff
@@ -16,3 +16,34 @@ INSERT INTO unit VALUES
 
 INSERT INTO `report` (`report_key`, `title_key`) VALUES
   ('stock_changes', 'REPORT.STOCK_CHANGES.TITLE');
+
+
+/*
+ * @author: jmcameron
+ * @date: 2020-11-23
+ * @pull-release: TBD
+ * @description: Rename reports to use underscores
```
Comment (jniles) [inline server/models/migrations/next/migrate.sql:23]: Excellent.

**Review by jniles — COMMENTED** [review]
I think this is great from a code perspective.  Let me give it a proper test tonight and I'll let you know if everything works.

**Comment by jniles** [comment]
LGTM.

bors r+

**Pull request merged**